lldb/source/Plugins/SymbolFile/DWARF/DWARFASTParser.cpp

```diff
@@ -56,8 +56,19 @@ DWARFASTParser::ParseChildArrayInfo(const DWARFDIE &parent_die,
                 if (auto frame = exe_ctx->GetFrameSP()) {
                   Status error;
                   lldb::VariableSP var_sp;
-                  auto valobj_sp = frame->GetValueForVariableExpressionPath(
-                      var_die.GetName(), eNoDynamicValues, 0, var_sp, error);
+                  lldb::TargetSP target_sp = frame->CalculateTarget();
+                  bool use_DIL = target_sp->GetUseDIL(
+                      (lldb_private::ExecutionContext *)exe_ctx);
```

Comment:
It looks like this cast wouldn't be required if you changed `GetExperimentalPropertyValue` to take a const execution context (all of the functions below it already take a const argument)

Reply:
This code & cast are gone now.